addons/mass_mailing/views/mailing_list_views.xml

```diff
@@ -108,131 +108,104 @@
         <field name="model">mailing.list</field>
         <field name="arch" type="xml">
             <kanban class="o_kanban_mobile o_kanban_mailing_list" sample="1">
-                <field name="name"/>
-                <field name="contact_count"/>
-                <field name="contact_count_email"/>
-                <field name="contact_count_opt_out"/>
-                <field name="contact_count_blacklisted"/>
-                <field name="contact_pct_bounce"/>
-                <field name="contact_pct_opt_out"/>
-                <field name="contact_pct_blacklisted"/>
                 <field name="active"/>
                 <templates>
-                    <t t-name="kanban-box">
+                    <t t-name="kanban-card">
                         <!-- Card when the Kanban view is grouped  -->
-                        <div class="oe_kanban_global_click o_mailing_list_kanban_grouped">
-                            <div class="oe_kanban_content d-flex flex-column h-100">
-                                <h2 class="mb-3 o_text_overflow">
-                                    <field name="name"/>
-                                </h2>
-                                <div class="d-flex align-items-center">
-                                    <div class="me-3">
-                                        <button class="btn btn-primary" name="action_view_contacts" type="object">
-                                            <t t-esc="record.contact_count.value"/> <span>Contacts</span>
-                                        </button>
+                        <div class="o_mailing_list_kanban_grouped d-flex flex-column h-100">
+                            <field name="name" class="fw-bold fs-2 mb-3 text-truncate"/>
+                            <div class="d-flex align-items-center">
+                                <button class="btn btn-primary me-3" name="action_view_contacts" type="object">
+                                    <field name="contact_count"/> <span>Contacts</span>
+                                </button>
+                                <div class="flex-grow-1 d-flex flex-column align-items-end o_mass_mailing_kanban_contact_links">
+                                    <a name="action_view_contacts_email" type="object">
+                                        <span>Valid Email Recipients</span>
+                                        <field name="contact_count_email" class="ms-3"/>
+                                    </a>
+                                </div>
+                            </div>
+                            <div class="flex-grow-1 d-flex align-items-end mt-4">
```

Comment:
are this div and the next one both necessary?

Reply:
No, remove the one div and add the necessary class to other div
Thank you.